.pre-commit-config.yaml

```diff
@@ -27,7 +27,7 @@ repos:
     hooks:
       - id: make-rst
         name: make-rst
-        entry: python3 doc/tools/make_rst.py doc/classes modules platform --dry-run --color
```

Comment:
This won't work on stock macOS, since only `python3` is part of the default installation (homebrew and similar package managers usually have `python` symlink as well).